src/js/lib/Scrollspy.jsx

```diff
@@ -142,36 +148,42 @@ export class Scrollspy extends React.Component {
   }
 
   _handleSpy () {
-    let timer
```

Comment:
In review of this I'm not sure how it could work. It's function scoped so every time we used the function it would start undefined.